server/pkg/git/run.go

```
@@ -0,0 +1,338 @@
+	return nil
+}
+
+type gitOutput struct {
+	stdout string
+	stderr string
+	code   int
+	err    error
+}
+
+func runGitCmd(dir string, args ...string) gitOutput {
+	cmd := exec.Command("git", args...)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)